controllers/infra/drivers/aws/create_volume.go

```
@@ -48,9 +49,7 @@ func (d Driver) createAndAttachVolumes(infra *v1.Infra, host *v1.Hosts, disks []
 }
 
 func (d Driver) createAndAttachVolume(infra *v1.Infra, host *v1.Hosts, disk *v1.Disk) error {
-	if disk.Name == "" {
-		return nil
-	}
+	deviceName, _ := generateDataDiskDeviceName(disk.Index)
```

should not ignore error